docs/reference/migration/migrate_6_3.asciidoc

```diff
@@ -10,3 +10,10 @@
 must now be specified in the client settings instead.
 
 See {plugins}/repository-gcs-client.html#repository-gcs-client[Google Cloud Storage Client Settings].
+
+==== Ingest Geoip Plugin
+
+* In earlier versions, database files have been stored as gzip compressed files with the extension `.gz`.
```

Comment:
maybe add a line about why we did this? Always better to understand why one needs to change his ways.

Reply:
I added an explanation for the original behavior and also why we did the change.